mmhuman3d/data/data_structures/human_data.py

```diff
@@ -30,39 +30,57 @@
     'bbox_xywh': {
         'type': np.ndarray,
         'shape': (-1, 5),
-        'temporal_dim': 0
+        'slice_dim': 0
     },
     'config': {
         'type': str,
+        'slice_dim': -1
     },
     'keypoints2d': {
         'type': np.ndarray,
         'shape': (-1, -1, 3),
-        'temporal_dim': 0
+        'slice_dim': 0
     },
     'keypoints3d': {
         'type': np.ndarray,
         'shape': (-1, -1, 4),
-        'temporal_dim': 0
+        'slice_dim': 0
     },
     'smpl': {
         'type': dict,
+        'slice_key': 'betas',
+        'slice_dim': 0
+    },
+    'smplh': {
+        'type': dict,
+        'slice_key': 'betas',
+        'slice_dim': 0
     },
     'smplx': {
         'type': dict,
+        'slice_key': 'betas',
```

Comment:
Good point. As the `sliced_key` is used to report data length in dict(instead of `len(dict)`) to HumanData, it must be exclusive. Considering data in `tests/data` for test_data_converters,` betas` is the only key passed the test.
After updating test data, we could select a better key than `betas`.